--- a/browser/branding/MozillaFirefox/brand.ftl
+++ b/browser/branding/MozillaFirefox/brand.ftl
@@ -1,7 +1,7 @@
-brand-short-name = Firefox
-brand-full-name = Mozilla Firefox
-brand-copy = Mozilla is a global, non-profit community dedicated to making the Web a better place.
-brand-third-party-info = Parts of this software are © 1998-2023 by individual contributors.
+brand-short-name = ArttulCat
+brand-full-name = ArttulCat by ArttulOS
+brand-copy = ArttulCat is a browser by ArttulOS, dedicated to a better browsing experience.
+brand-third-party-info = Portions of this software are © 1998-2023 by individual contributors.
 brand-eula-text = View the Mozilla Firefox End-User License Agreement
 brand-end-user-license-url = https://www.mozilla.org/legal/licenses/firefox-eula/
 brand-privacy-policy-url = https://www.mozilla.org/privacy/firefox/
--- a/browser/branding/MozillaFirefox/brand.js
+++ b/browser/branding/MozillaFirefox/brand.js
@@ -1,5 +1,5 @@
 pref("toolkit.defaultChromeURI", "chrome://browser/content/browser.xul");
-pref("browser.branding.vendor", "Mozilla");
-pref("browser.branding.userContext", "Firefox");
-pref("app.update.channel", "release");
+pref("browser.branding.vendor", "ArttulOS");
+pref("browser.branding.userContext", "ArttulCat");
+pref("app.update.channel", "arttulcat");
 pref("app.update.autoInstallEnabled", true);
--- a/browser/installer/windows/nsis/defines.nsi.in
+++ b/browser/installer/windows/nsis/defines.nsi.in
@@ -20,9 +20,9 @@
 
 !define PRODUCT_VENDOR_FULLNAME "@MOZILLA_VENDOR_FULLNAME@"
 !define PRODUCT_VENDOR_HOMEPAGE "@MOZILLA_VENDOR_HOMEPAGE@"
-!define PRODUCT_FULLNAME "Mozilla Firefox"
-!define PRODUCT_SHORTNAME "Firefox"
-!define PRODUCT_VENDOR "Mozilla"
+!define PRODUCT_FULLNAME "ArttulCat by ArttulOS"
+!define PRODUCT_SHORTNAME "ArttulCat"
+!define PRODUCT_VENDOR "ArttulOS"
 !define PRODUCT_LOGO_VERTICAL "@MOZ_BRAND_DIRECTORY@\vertical-s.ico"
 !define PRODUCT_ICON_PATH "${PRODUCT_BRAND_DIRECTORY}\browser\installer\windows\nsis\@MOZ_BRAND_SHORTNAME@.ico"
 
--- a/browser/locales/en-US/browser/appmenu.ftl
+++ b/browser/locales/en-US/browser/appmenu.ftl
@@ -2,4 +2,4 @@
 # The accessible label for the Firefox application menu button.
 # The menu button is found in the toolbar.
 # It represents the application menu.
-appmenu-accessible-label = Firefox Application Menu
+appmenu-accessible-label = ArttulCat Application Menu
--- a/browser/locales/en-US/browser/browser.ftl
+++ b/browser/locales/en-US/browser/browser.ftl
@@ -34,7 +34,7 @@
 #
 # Parameters:
 #   $title (String) - the title of the tab that was closed.
-window-title-tab-closed = { $title } - Mozilla Firefox
+window-title-tab-closed = { $title } - ArttulCat by ArttulOS
 
 # The label of the "Home" button.
 #
--- a/toolkit/xre/nsXULAppAPI.h
+++ b/toolkit/xre/nsXULAppAPI.h
@@ -19,8 +19,8 @@
 #ifndef nsXULAppAPI_h__
 #define nsXULAppAPI_h__
 
-#define XUL_APP_VENDOR_DIR_NAME "Mozilla"
-#define XUL_APP_NAME "firefox"
+#define XUL_APP_VENDOR_DIR_NAME "ArttulOS" // Changes default profile directory from ".mozilla" to ".arttulos"
+#define XUL_APP_NAME "arttulcat" // Changes executable name (e.g., firefox.exe to arttulcat.exe)
 
 #include "mozilla/Attributes.h"
 
