--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -1071,11 +1071,11 @@
 // The search engine to be used in the URL bar, search bar, and home page.
 // "defaultenginename" is the default for all regions.
 pref("browser.search.defaultenginename", "Google");
-// If "defaultenginename" is overridden by a regional preference (e.g.,
-// is in a region-specific override.ini or search.json.mozlz4), then this
-// preference defines which default engine is used for regions that don't
-// have a specified default engine.
-pref("browser.search.defaultenginename.US", "Google");
-pref("browser.search.order.US", "Google");
+// Replaced with Canine Tools. If "defaultenginename" is overridden by a regional
+// preference (e.g., is in a region-specific override.ini or search.json.mozlz4),
+// then this preference defines which default engine is used for regions that
+// don't have a specified default engine.
+pref("browser.search.defaultenginename.US", "Canine Tools");
+pref("browser.search.order.US", "Canine Tools");
 
 pref("browser.startup.homepage_override.mstone", "86.0");
--- a/browser/branding/MozillaFirefox/brand.ftl
+++ b/browser/branding/MozillaFirefox/brand.ftl
@@ -1,7 +1,7 @@
-brand-short-name = Firefox
-brand-full-name = Mozilla Firefox
-brand-copy = Mozilla is a global, non-profit community dedicated to making the Web a better place.
-brand-third-party-info = Parts of this software are © 1998-2023 by individual contributors.
+brand-short-name = ArttulCat
+brand-full-name = ArttulCat Browser
+brand-copy = ArttulCat is a community project dedicated to a better browsing experience.
+brand-third-party-info = Portions of this software are © 1998-2023 by individual contributors.
 brand-eula-text = View the Mozilla Firefox End-User License Agreement
 brand-end-user-license-url = https://www.mozilla.org/legal/licenses/firefox-eula/
 brand-privacy-policy-url = https://www.mozilla.org/privacy/firefox/
--- a/browser/branding/MozillaFirefox/brand.js
+++ b/browser/branding/MozillaFirefox/brand.js
@@ -1,5 +1,5 @@
 pref("toolkit.defaultChromeURI", "chrome://browser/content/browser.xul");
-pref("browser.branding.vendor", "Mozilla");
-pref("browser.branding.userContext", "Firefox");
-pref("app.update.channel", "release");
+pref("browser.branding.vendor", "ArttulCat");
+pref("browser.branding.userContext", "ArttulCat");
+pref("app.update.channel", "arttulcat");
 pref("app.update.autoInstallEnabled", true);
--- a/browser/components/search/resources/search.json
+++ b/browser/components/search/resources/search.json
@@ -3,29 +3,17 @@
 {
   "_locales": {
     "en-US": {
       "engines": [
         {
-          "name": "Wikipedia (en)",
-          "queryUrl": "https://en.wikipedia.org/wiki/Special:Search?search={searchTerms}",
-          "iconUrl": "resource://search-extensions/wikipedia@search.mozilla.org/icon.png"
-        },
-        {
-          "name": "Amazon.com",
-          "queryUrl": "https://www.amazon.com/s?field-keywords={searchTerms}&tag=firefox-20",
-          "iconUrl": "resource://search-extensions/amazon@search.mozilla.org/icon.png"
-        },
-        {
-          "name": "DuckDuckGo",
-          "queryUrl": "https://duckduckgo.com/?q={searchTerms}",
-          "iconUrl": "resource://search-extensions/ddg@search.mozilla.org/icon.png"
-        },
-        {
-          "name": "Google",
-          "queryUrl": "https://www.google.com/search?q={searchTerms}",
-          "iconUrl": "resource://search-extensions/google@search.mozilla.org/icon.png"
-        },
-        {
-          "name": "Bing",
-          "queryUrl": "https://www.bing.com/search?q={searchTerms}",
-          "iconUrl": "resource://search-extensions/bing@search.mozilla.org/icon.png"
+          "name": "Canine Tools",
+          "queryUrl": "https://search.canine.tools/search?q={searchTerms}",
+          "iconUrl": "https://search.canine.tools/favicon.ico",
+          "hidden": false,
+          "orderHint": 0,
+          "alias": "@canine",
+          "description": "The private search engine",
+          "tags": ["canine", "private", "search"],
+          "isUser-created": false
         }
       ]
     }
--- a/browser/installer/windows/nsis/defines.nsi.in
+++ b/browser/installer/windows/nsis/defines.nsi.in
@@ -20,9 +20,9 @@
 
 !define PRODUCT_VENDOR_FULLNAME "@MOZILLA_VENDOR_FULLNAME@"
 !define PRODUCT_VENDOR_HOMEPAGE "@MOZILLA_VENDOR_HOMEPAGE@"
-!define PRODUCT_FULLNAME "Mozilla Firefox"
-!define PRODUCT_SHORTNAME "Firefox"
-!define PRODUCT_VENDOR "Mozilla"
+!define PRODUCT_FULLNAME "ArttulCat Browser"
+!define PRODUCT_SHORTNAME "ArttulCat"
+!define PRODUCT_VENDOR "ArttulCat"
 !define PRODUCT_LOGO_VERTICAL "@MOZ_BRAND_DIRECTORY@\vertical-s.ico"
 !define PRODUCT_ICON_PATH "${PRODUCT_BRAND_DIRECTORY}\browser\installer\windows\nsis\@MOZ_BRAND_SHORTNAME@.ico"
 
--- a/browser/locales/en-US/browser/appmenu.ftl
+++ b/browser/locales/en-US/browser/appmenu.ftl
@@ -2,4 +2,4 @@
 # The accessible label for the Firefox application menu button.
 # The menu button is found in the toolbar.
 # It represents the application menu.
-appmenu-accessible-label = Firefox Application Menu
+appmenu-accessible-label = ArttulCat Application Menu
--- a/browser/locales/en-US/browser/browser.ftl
+++ b/browser/locales/en-US/browser/browser.ftl
@@ -34,7 +34,7 @@
 #
 # Parameters:
 #   $title (String) - the title of the tab that was closed.
-window-title-tab-closed = { $title } - Mozilla Firefox
+window-title-tab-closed = { $title } - ArttulCat Browser
 
 # The label of the "Home" button.
 #
--- a/toolkit/xre/nsXULAppAPI.h
+++ b/toolkit/xre/nsXULAppAPI.h
@@ -19,8 +19,8 @@
 #ifndef nsXULAppAPI_h__
 #define nsXULAppAPI_h__
 
-#define XUL_APP_VENDOR_DIR_NAME "Mozilla"
-#define XUL_APP_NAME "firefox"
+#define XUL_APP_VENDOR_DIR_NAME "ArttulCat" // Changes default profile directory from ".mozilla" to ".arttulcat"
+#define XUL_APP_NAME "arttulcat" // Changes executable name (e.g., firefox.exe to arttulcat.exe)
 
 #include "mozilla/Attributes.h"
 
